Userland/Libraries/LibCore/Version.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021, Mahmoud Mandour <ma.mandourr@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/String.h>
+
+namespace Core {
+
+String const SERENITY_VERSION = "Version 1.0";
```

Comment:
Since this is a fixed string, you could make it a `constexpr StringView SERENITY_VERSION = "Version 1.0"sv;`